en/Editing and formatting/Basic formatting syntax.md

```diff
@@ -213,18 +213,26 @@ You can create an unordered list by adding a `-`, `*`, or `+` before the text.
 - Second list item
 - Third list item
 
-To create an ordered list, start each line with a number followed by a `.` symbol.
```

Comment:
We don't encourage using parenthesis, see https://www.markdownguide.org/basic-syntax/#ordered-list-best-practices